omicron/parameters.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, version=None, defaults=dict(), **kwargs):
         if version is None:
             try:
                 version = utils.get_omicron_version()
-            except KeyError:
+            except (KeyError, RuntimeError):
```

Comment:
The `get_omicron_version` function, as far as I can tell, **cannot** raise a `KeyError` under any circumstances, so this should be a replacement rather than an append:

```suggestion
            except RuntimeError:
```

Reply:
However, I'm not sure that we should be skipping this sort of error anyway.

Reply:
When `omicron version` segfaults subprocess produces a runtime error.
Other omicron functions work.